### PR TITLE
Added TMTR_TMVeloxF7_config

### PR DESCRIPTION
**Pull request by tobeornottobe81**

### CODE REVIEW

File: configs/default/TMTR-TMVeloxF7.config [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Betaflight / STM32F7X2 (S7X2) 4.2.5 Nov 22 2020 / 18:39:53 (afdac08b3) MSP API: 1.43
+board_name TMVeloxF7
```
Comment (mikeller) [inline configs/default/TMTR-TMVeloxF7.config:2]: There are places like OSD that cannot display lower case characters. Please use only upper case for board names.

File: configs/default/TMTR-TMVeloxF7.config [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+dma pin A02 0
+# pin A02: DMA1 Stream 0 Channel 6
+dma pin A03 1
+# pin A03: DMA1 Stream 3 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature TELEMETRY
+feature RSSI_ADC
```
Comment (mikeller) [inline configs/default/TMTR-TMVeloxF7.config:110]: Does this board come with an on-board RX that uses this outdated way to receive an RSSI value?
If not, this should not be pre-configured.

File: configs/default/TMTR-TMVeloxF7.config [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set adc_device = 3
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
```
Comment (mikeller) [inline configs/default/TMTR-TMVeloxF7.config:126]: Is this an all-in-one offering that has the ESCs on board? If not this may not be used.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/TMTR-TMVeloxF7.config (hunk 1 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMVeloxF7.config:2]: OK, I will correct it

File: configs/default/TMTR-TMVeloxF7.config (hunk 2 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMVeloxF7.config:110]: Yes, the target board has RSSI function

Comment (mikeller) [reply configs/default/TMTR-TMVeloxF7.config:110]: What type of RX is on board on this flight controller - there probably are better way to get RSSI information than wasting an ADC input on an inaccurate way to transmit this information.

Comment (tobeornottobe81) [reply configs/default/TMTR-TMVeloxF7.config:110]: Analog RSSI signal from receiver, ADC is idle when not in use

File: configs/default/TMTR-TMVeloxF7.config (hunk 3 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMVeloxF7.config:126]: Yes, all of our ESC support dshot600

File: configs/default/TMTR-TMVeloxF7.config (hunk 2 above)
Comment (mikeller) [reply configs/default/TMTR-TMVeloxF7.config:110]: Why do you want to pre-select an outdated and poorly working option that is only needed for deprecated RX models, unless you had such a deprecated RX as part of your design? All current RX models support better ways to report RSSI.

File: configs/default/TMTR-TMVeloxF7.config (hunk 3 above)
Comment (mikeller) [reply configs/default/TMTR-TMVeloxF7.config:126]: So the user cannot connect this flight controller to a non - TMotor ESC. Can you elaborate a bit on how you achieve this?

File: configs/default/TMTR-TMVeloxF7.config (hunk 2 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMVeloxF7.config:110]: OK, we will change the design and remove the RSSI_ ADC

When F3 flight control was popular, our OSD adopted atmega328 + at7456 scheme, but the original function was retained here. Now it doesn't seem necessary

File: configs/default/TMTR-TMVeloxF7.config (hunk 3 above)
Comment (tobeornottobe81) [reply configs/default/TMTR-TMVeloxF7.config:126]: At present, market ESC support PWM, oneshot125,dshot protocol and so on, and can automatically identify.

**Review by mikeller — APPROVED** [review]

**Pull request merged**